src/Protocol/Diaspora.php

```diff
@@ -99,6 +99,53 @@ public static function relayList()
 		return $relay;
 	}
 
+	/**
+	 * @brief Return a list of participating contacts for a thread
+	 *
+	 * This is used for the participation feature.
+	 * One of the parameters is a contact array.
+	 * This is done to avoid duplicates.
+	 *
+	 * @param integer $thread   The id of the thread
+	 * @param array   $contacts The previously fetched contacts
+	 *
+	 * @return array of relay servers
+	 */
+	public static function participationsForThread($thread, $contacts)
```

Comment:
Is this method returning participations or participants? I think it should be explicit in the method name.

Reply:
it is always good having some (mostly) native speaker on board.